### PR TITLE
Added Norwegian language

### PR DESCRIPTION
**Pull request by dsbilling**

### CODE REVIEW

File: src/lang/no_NB.lang.php [hunk 1]
```diff
@@ -0,0 +1,366 @@
...
+ * PHP Server Monitor is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with PHP Server Monitor.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * @package     phpservermon
+ * @author      Pepijn Over <pep@mailbox.org>
```
Comment (TimZ99) [inline src/lang/no_NB.lang.php]: Aren't you the author of this file?

File: src/lang/no_NB.lang.php [hunk 2]
```diff
@@ -0,0 +1,366 @@
...
+		'delete_message' => 'Er du sikker på at du vil slette <b>alle</ b> logger?',
+	),
+	'servers' => array(
+		'server' => 'Server',
+		'status' => 'Status',
+		'label' => 'Label',
+		'domain' => 'Domene/IP',
+		'timeout' => 'Timeout',
+		'timeout_description' => 'Antall sekunder å vente på at serveren skal svare.',
+		'authentication_settings' => 'Autentiseringsinnstillinger (Optional)',
```
Comment (TimZ99) [inline src/lang/no_NB.lang.php]: Does 'optional' have a translation?

File: src/lang/no_NB.lang.php [hunk 3]
```diff
@@ -0,0 +1,366 @@
...
+		'log_sms' => 'Logg tekstmeldinger sendt av skriptet',
+		'log_pushover' => 'Logg pushover-meldinger sendt av skriptet',
+		'log_telegram' => 'Logg Telegram-meldinger sendt av skriptet',
+		'updated' => 'Konfigurasjonen er oppdatert.',
+		'tab_email' => 'E-post',
+		'tab_sms' => 'SMS',
+		'tab_pushover' => 'Pushover',
+		'tab_telegram' => 'Telegram',
+		'settings_email' => 'E-post innstillinger',
+		'settings_sms' => 'tekstmeldinger innstillinger',
```
Comment (TimZ99) [inline src/lang/no_NB.lang.php]: tekstmeldinger with a capital letter please

Comment (TimZ99) [reply src/lang/no_NB.lang.php]: SMS is used everywhere. Why do you use tekstmeldinger in this case?

File: src/lang/no_NB.lang.php [hunk 4]
```diff
@@ -0,0 +1,366 @@
...
+		'off_sms' => 'Serveren \'%LABEL%\' er NEDE: ip=%IP%, port=%PORT%. Feil=%ERROR%',
+		'off_email_subject' => 'VIKTIG: Serveren \'%LABEL%\' er NEDE',
+		'off_email_body' => "Kunne ikke koble til følgende server:<br/><br/>Server: %LABEL%<br/>IP: %IP%<br/>Port: %PORT%<br/>Feil: %ERROR%<br/>Dato: %DATE%",
+		'off_pushover_title' => 'Server \'%LABEL%\' er NEDE',
+		'off_pushover_message' => "Kunne ikke koble til følgende server:<br/><br/>Server: %LABEL%<br/>IP: %IP%<br/>Port: %PORT%<br/>Feil: %ERROR%<br/>Dato: %DATE%",
+		'off_telegram_message' => "Kunne ikke koble til følgende server:<br/><br/>Server: %LABEL%<br/>IP: %IP%<br/>Port: %PORT%<br/>Feil: %ERROR%<br/>Dato: %DATE%",
+		'on_sms' => 'Server \'%LABEL%\' er OPPE igjen: ip=%IP%, port=%PORT%, det var nede i %LAST_OFFLINE_DURATION%',
+		'on_email_subject' => 'VIKTIG: Server \'%LABEL%\' is RUNNING',
+		'on_email_body' => "Server '%LABEL%' er oppe igjen, det var nede i %LAST_OFFLINE_DURATION%:<br/><br/>Server: %LABEL%<br/>IP: %IP%<br/>Port: %PORT%<br/>Dato: %DATE%",
+		'on_pushover_title' => 'Server \'%LABEL%\' is RUNNING',
```
Comment (TimZ99) [inline src/lang/no_NB.lang.php]: Forgotten?

File: src/lang/en_US.lang.php [hunk 5]
```diff
@@ -306,7 +306,7 @@
 		'test_message' => 'Test message',
 		'email_sent' => 'Email sent',
 		'email_error' => 'Error in email sending',
-		'sms_sent' => 'SMS sent',
+		'sms_sent' => 'Sms sent',
```
Comment (TimZ99) [inline src/lang/en_US.lang.php]: SMS is used everywhere else in this file. What is the reason that this one needs to be changed?

**Review by TimZ99 — CHANGES_REQUESTED** [review]
Thanks for the translation! No major issues, just a few questions, and some changes.

**Comment by dsbilling** [comment]
Fixed the issues :)

File: src/lang/no_NB.lang.php [hunk 6]
```diff
@@ -0,0 +1,366 @@
...
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with PHP Server Monitor.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * @package     phpservermon
+ * @author      Daniel S. Billing <daniel@infihex.com>
+ * @copyright   Copyright (c) 2008-2018 Daniel S. Billing <daniel@infihex.com>
```
Comment (TimZ99) [inline src/lang/no_NB.lang.php]: Also the copyright?

**Comment by dsbilling** [comment]
Done 💯

**Review by TimZ99 — APPROVED** [review]

**Comment by TimZ99** [comment]
Thanks a lot! 🎉

**Pull request merged**